console/repositories/market_app_repo.py

```diff
@@ -92,6 +144,35 @@ def _prepare_get_rainbond_app_by_query_sql(self,
             eid=eid, extend_where=extend_where, offset=(page - 1) * page_size, rows=page_size, join_version=join_version)
         return sql
 
+    def get_rainbond_plugin_app_in_teams_by_querey(self,
```

Comment:
`get_rainbond_plugin_app_in_teams_by_querey` Misspelled, it should be`get_rainbond_plugin_app_in_teams_by_query`